regression/goto-instrument/replace-function-body-struct-with-const-member/main.c

```diff
@@ -0,0 +1,21 @@
+#include <assert.h>
+
+struct WithConstMember
+{
+  int non_const;
+  const int is_const;
+};
+
+struct WithConstMember globalStruct = {10, 20};
+void havoc_struct(struct WithConstMember *s);
+
+int main(void)
+{
+  struct WithConstMember paramStruct = {11, 21};
+  havoc_struct(&paramStruct);
+  assert(globalStruct.non_const == 10);
+  assert(globalStruct.is_const == 20);
+  assert(paramStruct.non_const == 11);
```

Comment:
Why is this not been havoced? (A good technique for trying to verify these tests is to try and write bodies that violate this asserts, for example for this can you not do:

```C
(*WithConstMember).non_const = 20;
```